ui/helpers/utils/notification.utils.test.ts

```diff
@@ -9,7 +9,7 @@ import {
 describe('formatMenuItemDate', () => {
   beforeAll(() => {
     jest.useFakeTimers();
-    jest.setSystemTime(new Date('2024-06-07T09:40:00Z'));
+    jest.setSystemTime(new Date(Date.UTC(2024, 5, 7, 9, 40, 0))); // 2024-06-07T09:40:00Z
```

Comment:
This is the main culprit. We were setting the system time to a TZ specific date.

We have now changed this to a UTC (timezone agnostic) date. Hopefully this resolves issues that devs may face when running this test locally.